core/templates/dev/head/pages/question_player/QuestionPlayerDirective.ts

```diff
@@ -79,11 +79,11 @@ oppia.directive('questionPlayer', [
         '/pages/question_player/question_player_directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$rootScope', 'QuestionPlayerBackendApiService',
+        '$rootScope', '$scope', 'QuestionPlayerBackendApiService',
```

Comment:
Sync with upstream/develop, this fix should be in

Reply:
Done.

Reply:
That's weird. I merged it with develop branch, but it still shows up in code diff.